### PR TITLE
Add line-height-step to properties.json.

### PR DESCRIPTION
**Pull request by jpmedley**
This isn't quite ready. I wasn't able to figure out what belongs in the animationType field.

### CODE REVIEW

**Comment by jpmedley** [comment]
The spec says that this property is [not even animatable](https://drafts.csswg.org/css-rhythm/#line-height-step). So, it can't have any of the animation types listed in the schema. Can someone please tell me how to proceed?

**Comment by Elchi3** [comment]
(I'm not a CSS expert nor an active maintainer of the MDN CSS docs, but ...)

I think it should be "discrete". It's shame we don't explain this on the MDN CSS pages. There should be glossary entry explaining discrete animations and every mention of "discrete" should link to it.
c.f. https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat
https://stackoverflow.com/questions/44510663/what-are-discrete-animations

I think this came from some specs changing "Animatable" to "Animation type" but there are still some specs around that use "Animatable". On MDN we use "Animation type" everywhere I think.

**Comment by jpmedley** [comment]
Done. PTAL.

File: css/properties.json [hunk 1]
```diff
@@ -4094,6 +4094,21 @@
...
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "none",
+    "appliesto": "blockContainerElements",
+    "computed": "absoluteLength0ForNone",
+    "order": "perGrammar",
+    "status": "nonstandard"
```
Comment (Elchi3) [inline css/properties.json]: I guess this should be "standard" as there is a specification for this.

Comment (jpmedley) [inline css/properties.json]: Yes https://drafts.csswg.org/css-rhythm/#line-height-step

But I thought 'standard' was reserved for things that are full fledged recommendations.

Comment (Elchi3) [inline css/properties.json]: Sorry, but I would say definitely not "nonstandard". I guess "standard" is the default/standard setting I would set here. But possibly "experimental" (I'm not always clear on how we use that term, and I look at other usages of it to judge.)

Other properties that have "experimental": display-list, display-outside, font-variation-settings, initial-letter, max-block-size, offset, ... What do you think?

Comment (jpmedley) [inline css/properties.json]: My sense is that 'experimental' would be more consistent with what you guys typically do. Changing it now.

**Pull request merged**